### ISSUE TITLE
Nondeterministic failure of core_vnode_eqc:prop_simple()

### ISSUE THREAD

**Issue by slfritchie** [basho/riak_core#253]
I can't tell if this is due to the QC model not taking wall-clock time into account or if this is a real bug.

To reproduce:

```
% make test
% erl -pz .eunit deps/*/ebin
> C1 = [[{init,{setup,{qcst,[],[],undefined,[0,182687704666362864775460604089535377456991567872,365375409332725729550921208179070754913983135744,548063113999088594326381812268606132370974703616,730750818665451459101842416358141509827966271488,913438523331814323877303020447676887284957839360,1096126227998177188652763624537212264741949407232,1278813932664540053428224228626747642198940975104],[],false,0,[]}}},{set,{var,1},{call,core_vnode_eqc,prepare,[0]}},{set,{var,2},{call,core_vnode_eqc,start_vnode,[0]}},{set,{var,3},{call,mock_vnode,crash,[{0,nonode@nohost}]}},{set,{var,4},{call,mock_vnode,neverreply,[[{0,nonode@nohost}]]}},{set,{var,5},{call,mock_vnode,neverreply,[[{0,nonode@nohost}]]}},{set,{var,6},{call,mock_vnode,stop,[{0,nonode@nohost}]}},{set,{var,7},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},{set,{var,8},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},{set,{var,9},{call,core_vnode_eqc,start_vnode,[182687704666362864775460604089535377456991567872]}},{set,{var,10},{call,mock_vnode,get_counter,[{182687704666362864775460604089535377456991567872,nonode@nohost}]}},{set,{var,11},{call,mock_vnode,get_index,[{182687704666362864775460604089535377456991567872,nonode@nohost}]}},{set,{var,12},{call,mock_vnode,crash,[{182687704666362864775460604089535377456991567872,nonode@nohost}]}},{set,{var,13},{call,core_vnode_eqc,start_vnode,[730750818665451459101842416358141509827966271488]}},{set,{var,14},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},{set,{var,15},{call,core_vnode_eqc,latereply,[[{730750818665451459101842416358141509827966271488,nonode@nohost}]]}},{set,{var,16},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},{set,{var,17},{call,mock_vnode,get_crash_reason,[{730750818665451459101842416358141509827966271488,nonode@nohost}]}},{set,{var,18},{call,mock_vnode,get_counter,[{182687704666362864775460604089535377456991567872,nonode@nohost}]}},{set,{var,19},{call,core_vnode_eqc,start_vnode,[1278813932664540053428224228626747642198940975104]}},{set,{var,20},{call,mock_vnode,get_counter,[{182687704666362864775460604089535377456991567872,nonode@nohost}]}},{set,{var,21},{call,mock_vnode,get_index,[{182687704666362864775460604089535377456991567872,nonode@nohost}]}},{set,{var,22},{call,mock_vnode,get_index,[{1278813932664540053428224228626747642198940975104,nonode@nohost}]}},{set,{var,23},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},{set,{var,24},{call,core_vnode_eqc,latereply,[[{1278813932664540053428224228626747642198940975104,nonode@nohost}]]}}],[{res,[]}]].
> core_vnode_eqc:setup_simple().
> [true = eqc:check(core_vnode_eqc:prop_simple(), C1) || _ <- lists:seq(1,50*1000)].
```

After some period of time, I see:
    OK, passed the test.
    OK, passed the test.
    Failed! 
    [{init,{setup,{qcst,[],[],undefined,
                        [0,182687704666362864775460604089535377456991567872,
                         365375409332725729550921208179070754913983135744,
                         548063113999088594326381812268606132370974703616,
                         730750818665451459101842416358141509827966271488,
                         913438523331814323877303020447676887284957839360,
                         1096126227998177188652763624537212264741949407232,
                         1278813932664540053428224228626747642198940975104],
                        [],false,0,[]}}},
     {set,{var,1},{call,core_vnode_eqc,prepare,[0]}},
     {set,{var,2},{call,core_vnode_eqc,start_vnode,[0]}},
     {set,{var,3},{call,mock_vnode,crash,[{0,nonode@nohost}]}},
     {set,{var,4},{call,mock_vnode,neverreply,[[{0,nonode@nohost}]]}},
     {set,{var,5},{call,mock_vnode,neverreply,[[{0,nonode@nohost}]]}},
     {set,{var,6},{call,mock_vnode,stop,[{0,nonode@nohost}]}},
     {set,{var,7},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},
     {set,{var,8},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},
     {set,{var,9},
          {call,core_vnode_eqc,start_vnode,
                [182687704666362864775460604089535377456991567872]}},
     {set,{var,10},
          {call,mock_vnode,get_counter,
                [{182687704666362864775460604089535377456991567872,
                  nonode@nohost}]}},
     {set,{var,11},
          {call,mock_vnode,get_index,
                [{182687704666362864775460604089535377456991567872,
                  nonode@nohost}]}},
     {set,{var,12},
          {call,mock_vnode,crash,
                [{182687704666362864775460604089535377456991567872,
                  nonode@nohost}]}},
     {set,{var,13},
          {call,core_vnode_eqc,start_vnode,
                [730750818665451459101842416358141509827966271488]}},
     {set,{var,14},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},
     {set,{var,15},
          {call,core_vnode_eqc,latereply,
                [[{730750818665451459101842416358141509827966271488,
                   nonode@nohost}]]}},
     {set,{var,16},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},
     {set,{var,17},
          {call,mock_vnode,get_crash_reason,
                [{730750818665451459101842416358141509827966271488,
                  nonode@nohost}]}},
     {set,{var,18},
          {call,mock_vnode,get_counter,
                [{182687704666362864775460604089535377456991567872,
                  nonode@nohost}]}},
     {set,{var,19},
          {call,core_vnode_eqc,start_vnode,
                [1278813932664540053428224228626747642198940975104]}},
     {set,{var,20},
          {call,mock_vnode,get_counter,
                [{182687704666362864775460604089535377456991567872,
                  nonode@nohost}]}},
     {set,{var,21},
          {call,mock_vnode,get_index,
                [{182687704666362864775460604089535377456991567872,
                  nonode@nohost}]}},
     {set,{var,22},
          {call,mock_vnode,get_index,
                [{1278813932664540053428224228626747642198940975104,
                  nonode@nohost}]}},
     {set,{var,23},{call,riak_core_vnode_master,all_nodes,[mock_vnode]}},
     {set,{var,24},
          {call,core_vnode_eqc,latereply,
                [[{1278813932664540053428224228626747642198940975104,
                   nonode@nohost}]]}}]
    History: [{{setup,{qcst,[],[],undefined,
                            [0,182687704666362864775460604089535377456991567872,
                             365375409332725729550921208179070754913983135744,
                             548063113999088594326381812268606132370974703616,
                             730750818665451459101842416358141509827966271488,
                             913438523331814323877303020447676887284957839360,
                             1096126227998177188652763624537212264741949407232,
                             1278813932664540053428224228626747642198940975104],
                            [],false,0,[]}},
               <0.3891.0>},
              {{stopped,{qcst,[],[],<0.3891.0>,
                              [0,182687704666362864775460604089535377456991567872,
                               365375409332725729550921208179070754913983135744,
                               548063113999088594326381812268606132370974703616,
                               730750818665451459101842416358141509827966271488,
                               913438523331814323877303020447676887284957839360,
                               1096126227998177188652763624537212264741949407232,
                               1278813932664540053428224228626747642198940975104],
                              [],false,0,[]}},
               ok},
              {{running,{qcst,[0],
                              [{0,0}],
                              <0.3891.0>,
                              [0,182687704666362864775460604089535377456991567872,
                               365375409332725729550921208179070754913983135744,
                               548063113999088594326381812268606132370974703616,
                               730750818665451459101842416358141509827966271488,
                               913438523331814323877303020447676887284957839360,
                               1096126227998177188652763624537212264741949407232,
                               1278813932664540053428224228626747642198940975104],
                              [{0,undefined}],
                              false,0,[]}},
               ok},
              {{running,{qcst,[0],
                              [{0,0}],
                              <0.3891.0>,
                              [0,182687704666362864775460604089535377456991567872,
                               365375409332725729550921208179070754913983135744,
                               548063113999088594326381812268606132370974703616,
                               730750818665451459101842416358141509827966271488,
                               913438523331814323877303020447676887284957839360,
                               1096126227998177188652763624537212264741949407232,
                               1278813932664540053428224228626747642198940975104],
                              [{0,0}],
                              false,0,[]}},
               ok},
              {{running,{qcst,[0],
                              [{0,1}],
                              <0.3891.0>,
                              [0,182687704666362864775460604089535377456991567872,
                               365375409332725729550921208179070754913983135744,
                               548063113999088594326381812268606132370974703616,
                               730750818665451459101842416358141509827966271488,
                               913438523331814323877303020447676887284957839360,
                               1096126227998177188652763624537212264741949407232,
                               1278813932664540053428224228626747642198940975104],
                              [{0,0}],
                              false,0,[]}},
               ok},
              {{running,{qcst,[0],
                              [{0,2}],
                              <0.3891.0>,
                              [0,182687704666362864775460604089535377456991567872,
                               365375409332725729550921208179070754913983135744,
                               548063113999088594326381812268606132370974703616,
                               730750818665451459101842416358141509827966271488,
                               913438523331814323877303020447676887284957839360,
                               1096126227998177188652763624537212264741949407232,
                               1278813932664540053428224228626747642198940975104],
                              [{0,0}],
                              false,0,[]}},
               stopped},
              {{running,{qcst,[],
                              [{0,0}],
                              <0.3891.0>,
                              [0,182687704666362864775460604089535377456991567872,
                               365375409332725729550921208179070754913983135744,
                               548063113999088594326381812268606132370974703616,
                               730750818665451459101842416358141509827966271488,
                               913438523331814323877303020447676887284957839360,
                               1096126227998177188652763624537212264741949407232,
                               1278813932664540053428224228626747642198940975104],
                              [{0,undefined}],
                              false,0,[]}},
               []},
              {{running,{qcst,[],
                              [{0,0}],
                              <0.3891.0>,
                              [0,182687704666362864775460604089535377456991567872,
                               365375409332725729550921208179070754913983135744,
                               548063113999088594326381812268606132370974703616,
                               730750818665451459101842416358141509827966271488,
                               913438523331814323877303020447676887284957839360,
                               1096126227998177188652763624537212264741949407232,
                               1278813932664540053428224228626747642198940975104],
                              [{0,undefined}],
                              false,0,[]}},
               [<0.3896.0>,<0.3895.0>]}]
    State: {qcst,[],
                 [{0,0}],
                 <0.3891.0>,
                 [0,182687704666362864775460604089535377456991567872,
                  365375409332725729550921208179070754913983135744,
                  548063113999088594326381812268606132370974703616,
                  730750818665451459101842416358141509827966271488,
                  913438523331814323877303020447676887284957839360,
                  1096126227998177188652763624537212264741949407232,
                  1278813932664540053428224228626747642198940975104],
                 [{0,undefined}],
                 false,0,[]}
    Result: {postcondition,false}
    res: failed
    {postcondition,false} /= ok
    *\* exception error: no match of right hand side value false

**Comment by rzezeski** [comment]
Scott, I believe this was fixed in #260.  Can you please verify?

**Comment by slfritchie** [comment]
Hi, Ryan, sorry about ignoring this.  After several 10Ks of iterations on today's 'develop', I believe you're right.  Closing.
